andela/ah-backend-athena
# #162163276 Tag Articles

**What does this PR do?**
This pull request delivers a feature that enables an author to add tags while creating an article

**Description of the tasks to be completed**
Before this feature, an author could just create an article without summarizing what the writing is about with a tag. This feature delivers the option of the author adding a tag to kind of say what the write up is about.

**How can this be manually tested?**
Go to the endpoint for creating an article `/api/articles/` 


after posting you should see the following result
<img width="1117" alt="screenshot 2018-12-13 at 18 46 26" src="https://user-images.githubusercontent.com/39732510/49949846-b6b57780-ff07-11e8-8985-6f4b658350b5.png">

- You can also view tags associted with a partcular article by navigating to `/api/:article/tags/`
<img width="1128" alt="screenshot 2018-12-13 at 18 47 11" src="https://user-images.githubusercontent.com/39732510/49949853-b917d180-ff07-11e8-8d81-7318aefe20f7.png">
- You can also delete a tag by navigating to `/api/:article/<tag>/`


<img width="1122" alt="screenshot 2018-12-13 at 18 47 35" src="https://user-images.githubusercontent.com/39732510/49949860-bb7a2b80-ff07-11e8-8898-7c3148ebc796.png">
